py3status/modules/deadbeef.py

```diff
@@ -4,21 +4,23 @@
 
 Configuration parameters:
     cache_timeout: refresh interval for this module (default 1)
-    delimiter: delimiter between metadata fields (default '¥å¥å¥å')
-    format: display format for this module (default '{artist} - {title}')
+    delimiter: delimiter between metadata fields (default 'ÐÉÅÐƁÊËF')
+    format: display format for this module (default '[{artist} - ][{title}]')
 
 Format placeholders:
-    {artist}        artist
-    {length}        length
-    {playback_time} playback time
-    {title}         title
-    {tracknumber}   track number
-    {year}          year
+    {artist} Name of the artist of the track.
+    {isplaying} "1" if file is currently playing, empty string otherwise.
+    {length} The length of the track formatted as hours, minutes,
+                and seconds, rounded to the nearest second.
+    {playback_time} The elapsed time formatted as [HH:]MM:SS.
+    {title} Title of the track.
+    {tracknumber} Two-digit index of specified track within the album.
```

Comment:
Yes. We call it `track[n]umber` in old format... or `n`.
`{tracknum}` is a trimmed placeholder.

New format uses `{tracknumber}`. Could keep `{tracknum}`, but opt not to.
`{track number}` (w/ a space) comes with no prefix 2d. thing.